Docs/sp-imp-spec.txt

```diff
@@ -225,11 +225,11 @@
 
     sql_parse.cc:mysql_execute_command() then uses sp.cc:sp_find() to
     get the sp_head for the procedure (which may have been read from the
-    database or feetched from the in-memory cache) and calls the sp_head's
+    database or fetched from the in-memory cache) and calls the sp_head's
     method execute().
-    Note: It's important that substatements called by the procedure do not
+    Note: It's important that sub statements called by the procedure do not
```

Comment:
What's wrong about substatement? Seem to be exactly the right word: https://en.wiktionary.org/wiki/substatement

Reply:
Ya, you are right, that's my mistake, it's correct